man/systemd-timedated.service.xml

```diff
@@ -46,25 +46,36 @@
   </refsect1>
 
   <refsect1>
-    <title>Environment</title>
-
-    <variablelist class='environment-variables'>
-      <varlistentry>
-        <term><varname>$SYSTEMD_TIMEDATED_NTP_SERVICES</varname></term>
-
-        <listitem><para>Colon-separated list of unit names of NTP client services.
-        If not set, then
-        <citerefentry><refentrytitle>systemd-timesyncd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
-        is used. See the entries of NTP related commands of
-        <citerefentry><refentrytitle>timedatectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-        for details about this.</para>
-
-        <para>Example:
-        <programlisting>SYSTEMD_TIMEDATED_NTP_SERVICES=ntpd.service:chronyd.service:systemd-timesyncd.service</programlisting>
-        </para></listitem>
-      </varlistentry>
-    </variablelist>
+    <title>List of network time synchronization services</title>
+
+    <para><command>systemd-timesyncd</command> will look for files with a <literal>.list</literal> extension
+    in <filename>ntp-units.d/</filename> directories. Each file is parsed as a list of unit names, one per
+    line. Empty lines and lines with comments (<literal>#</literal>) are ignored.  Files are read from
+    <filename>/usr/lib/systemd/ntp-units.d/</filename> and the corresponding directories under
+    <filename>/etc/</filename>, <filename>/run/</filename>, <filename>/usr/local/lib/</filename>. Files in
+    <filename>/etc/</filename> override files with the same name in <filename>/run/</filename>,
+    <filename>/usr/local/lib/</filename>, and <filename>/usr/lib/</filename>. Files in
+    <filename>/run/</filename> override files with the same name under <filename>/usr/</filename>. Packages
+    should install their configuration files in <filename>/usr/lib/</filename> (distribution packages) or
+    <filename>/usr/local/lib/</filename> (local installs).</para>
```

Comment:
reading this was a bit confusing... you explain how to build the list of timesync services but you don't really say what they are used for anywhere. I assume that they become some sort of dependency for systemd-timesyncd but that's never explicitely stated

is there a generator that reads them at early boot, or are they added to the transaction ?